### DISCUSSION TITLE
Is it possible to make my lists private?

### DISCUSSION THREAD

**Discussion by ThallyssonKlein** [community/community#10750] [Other Feature Feedback, Questions, & Ideas] [answered]
I noticed that my lists are visible to other users and that's not cool. Is it possible to make them private?

**Comment by bailey** [comment]
Hi, thanks for reaching out @ThallyssonKlein! Lists, just like Stars themselves, are public-by-default right now. The exception here is that private repositories will only be visible to users who have access to the repository. So if your entire List is made up of private repositories, users without access to any of those repositories will just see an empty list.

I'm interested to hear more on why you'd prefer your lists to be private if you'd be willing to share some more details. And thanks again for this feedback. 👾

**Reply by kirby-enlightened** [reply]
If a list appears empty due to only being populated by private repos, **there should at least be some indication of the list containing only private repositories**, and this is why it may be a good idea to allow private lists.

**Reply by ThallyssonKlein** [reply]
@bailey I used stars to save useful repositories for some of my purposes and I was already thinking of a solution to categorize them and I thought it was great that Github implemented this, avoiding the work of making an extension or something to solve this problem. And given my use of stars, the lists for me are an identification of the goals I have with the repositories I've saved, so something that I don't want to share with others automatically but that I might want in some specific lists.

**Reply by skrish13** [reply]
Starred repos are like Liked Tweets. Hard to sift through thousands. But starred lists are like Twitter Lists which can be more effortful and private. If this feature isn't available, I'd just create a Notion list or something instead.

**Reply by mijailr** [reply]
The lists are useful to organize workstreams, so to have private lists could help to not organize private workstreams, in some scenarios this privacy is mandatory. So private list could help a lot in those scenarios.

**Reply by soulrx** [reply]
This seems to miss a chance for a nice github feature.  I just want to tag repos I run across that I find interesting for personal use later. Is there not a way to do that? Not public just a feature if I am logged in and browsing repos. Just a link on every repo if you are logged in like "private pin". Then a section that lets me look at my pins.

**Reply by pataquets** [reply]
I see them as a bookmarking device, which might have multiple use-cases (research, planning, stacks) which you might want to keep private.
I'm not using them as of now for such cases due to publicness.

**Reply by F153** [reply]
Go

**Reply by F153** [reply]
Go

**Reply by T9ipa** [reply]
Go to Home

**Reply by geekley** [reply]
> why you'd prefer your lists to be private

Many people care a lot about privacy and don't want to have to worry about side-effects of everything they do on the Internet. That's why lots of actions should follow the good design of privacy-by-default where ***consent for publishing anything should be completely unambiguous***. That is: only actions that are obviously public (such as commenting or clicking like) should be public - for those you already intend or at least expect that they'll be public.

Now with something like clicking a :star:, you could reasonably expect people to use it as a bookmark, assuming it's private; unaware that it's not. E.g. in Firefox, a star icon is used for bookmarks, which are obviously not public. It's probably the same in most sites.

In GitHub's case, at least it's not that bad because it shows the number of people who starred a repo, giving an indication it might be public; still that's not an inference everyone will make, as it could be the case that only the stat is public, not the list of people.

Of course, one could argue the majority of people don't care that much about privacy (to the point of going out of their way to consider it); or there might be some commercial interest in GitHub's end in "assuming" people want that data public.
Which, fair enough, that's fine... if people at least have the *option* to opt-out.
But I'd still say privacy-by-default is better consumer-oriented practice, specially if your consumers are mostly tech-savvy people.

**Reply by Mrk994** [reply]
OK this is a very big and very important part of my life and I believe that is more important for somewhere about 100 people have better everything because of me and my crazy idess so goodbye

**Reply by Alexander8186** [reply]
Я за обеими руками и ногами

**Comment by alexjalonso** [comment]
1. ``

- [ ] @### 

> _****_

**Comment by BELIEVENATION** [comment]
Yes

**Reply by Litlangelok** [reply]
> Yes

**Comment by tsp187** [comment]
> Hi, thanks for reaching out @ThallyssonKlein! Lists, just like Stars themselves, are public-by-default right now. The exception here is that private repositories will only be visible to users who have access to the repository. So if your entire List is made up of private repositories, users without access to any of those repositories will just see an empty list.
> 
> I'm interested to hear more on why you'd prefer your lists to be private if you'd be willing to share some more details. And thanks again for this feedback. 👾

Unified / follow path

**Reply by tsp187** [reply]
Enable Default

**Comment by Zi-Tao** [comment, marked as answer]
@ThallyssonKlein https://github.com/github-community/community/discussions/8766

**Comment by gyt95** [comment]
In fact, I hope to only hide my dynamic display on other people's homepage, but now only provides a beta version of the self-closing function —— Make profile private and hide activity.

**Comment by matiaslopezd** [comment]
Yes, please! 👍🏻

**Comment by amindadgar** [comment]
Have the same issue, it would be great to have the option to make my lists private to myself!

**Comment by ArkhamCookie** [comment]
I would love this too.

**Reply by mijailr** [reply]
?

**Comment by thinktapper** [comment]
I started utilizing the lists feature recently and was shocked to find out they'd been so public this whole time. Searched through all my GitHub settings looking for an option, and wound up here.

I've personally been using lists like I would playlists on YouTube → categories for different sources of information or a 'read-later' like bucket for specific personal projects.

Would love a private option here - but in the meantime I guess I'll go back to tossing bookmarks in messy browser folders :wastebasket:

**Comment by shinebayar-g** [comment]
It's unfortunate that it's public only. Someone pointed out Youtube as a nice example. I agree, they have multiple options like private, public, link only etc. If this is implemented, it would open new use cases like sharing my starred repositories with someone else, but not necessarily making them publicly available. For now, heading back to good old bookmarks.. 😮‍💨 

Someone [said](https://github.com/community/community/discussions/8811#discussioncomment-1804407) it's coming soon over a year ago ..

**Comment by kvedantmahajan** [comment]
Much needed. I have list of frontend interview questions and made a list for bookmarking similar repos. Embarrasing to discover this as every job application asks for Github link. They can choose to not ask what I have prepared for. Plus, I heavily use it as bookmarking facility

**Comment by mjraha592** [comment]
> Hi, thanks for reaching out @ThallyssonKlein! Lists, just like Stars themselves, are public-by-default right now. The exception here is that private repositories will only be visible to users who have access to the repository. So if your entire List is made up of private repositories, users without access to any of those repositories will just see an empty list.
> 
> I'm interested to hear more on why you'd prefer your lists to be private if you'd be willing to share some more details. And thanks again for this feedback. 👾

Hello, these days, you can't tell the truth from a lie, and the people and men around us have completely changed, and it seems that the doorman's friends are deceitful and taking advantage of each other has become a normal and everyday thing. I prefer to act privately and completely secretly.  I have complete trust in Wyman

**Comment by AbhimanyuAryan** [comment]
when is private list feature coming out? Its kinda useless right now

**Comment by AbhimanyuAryan** [comment]
@bailey who’s the project manager on this currently?

**Comment by Bodarobi** [comment]
Hallo sziasztok üdvözletem

**Comment by MarkOWiesemann** [comment]
I am guessing it is still not possible? @bailey

**Comment by 0918039995a** [comment]
是因為

**Comment by 0918039995a** [comment]
的

**Comment by Luckthrone** [comment]
Best fo you

**Comment by ishowman** [comment]
how to read my list. i can not find them

**Reply by Bodarobi** [reply]
**_[![image](https://github.com/community/community/assets/143372536/6f3be6a8-e67c-4308-9828-2b26a8e58d41)]()_**

**Reply by Bodarobi** [reply]
****_[]()_****

**Comment by Alexander8186** [comment]
[]()@🥰

**Comment by kiyanmr** [comment]
Good
